tests/testthat/test-matrix.r

```diff
@@ -201,23 +202,29 @@ test_that("Taking sample of estimated participant's give na when no overlap with
   expect_true(is.na(rowSums(cm$matrix)[4]))
   expect_false(is.na(rowSums(cm$matrix)[5]))
 })
-test_that("If weights = weigh.dayofweek, the results are identical", {
-  expect_identical(
-    suppressMessages(
-      contact_matrix(
-        survey = polymod10, countries = "United Kingdom",
-        weigh.dayofweek = TRUE
-      )
-    ),
-    suppressMessages(
-      contact_matrix(
-        survey = polymod10, countries = "United Kingdom",
-        weights = "added_weight"
+
+test_that("If weights = added_weight, the results are not identical", {
+  cm_orig <- suppressMessages(
+    contact_matrix(
+      survey = polymod10, countries = "United Kingdom",
+      age.limits = c(0, 18, 60),
+      return.part.weights = T
```

Comment:
```suggestion
      return.part.weights = TRUE
```

It's safer to use `TRUE` rather than `T` because `T` is not a reserved variable and can be overriden:

``` r
T <- FALSE

say_hello <- T

if (say_hello) {
  cat("Hello")
} else {
  cat("Bye")
}
#> Bye
```

<sup>Created on 2022-09-15 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>